README.md

```diff
@@ -13,25 +13,20 @@ if he is not logged in.
 /**
  * A sample default action that requires to be logged.
  *
- * @URL /homepage
+ * @URL ("/homepage")
  * @Logged
  */
 public function index() { ... }
 ```
 
-The <b>@Logged</b> annotation requires an instance of UserService to exist. The
-name of the instance must be "userService".
-If your UserService instance is not named "userService" (or if you want to use several UserService instances,
-you can specify the instance of UserService to use in parameter of the annotation:
+
 
 ```php
 /**
  * A sample default action that requires to be logged.
- * The "myUserService" Mouf instance will be used to check
- * whether the user is logged or not.
  *
- * @URL /homepage
- * @Logged("myUserService")
+ * @URL ("/homepage")
+ * @Logged("myUnauthorizedMidleware")
```

Comment:
myUnauthorizedMidleware => myUnauthorizedMiddleware

Also, maybe we want a key to be specified like:

@Logged("middlewareName"=>"myUnauthorizedMidleware")